lib/core/src/persist/mod.rs

```diff
@@ -110,6 +113,7 @@ impl Persister {
                     false => PaymentType::Send,
                 },
                 is_confirmed: is_tx_confirmed,
+                unblinding_data: Some(unblinding_data),
```

Comment:
Seems to me all Liquid txs are blinded (confidential), so why store this as optional?

Reply:
For the inserted pseudo txs we don't have this data